apps/explorer/src/pages/validator/ValidatorStats.tsx

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { DelegationAmount } from './DelegationAmount';
+
+import type { Validator } from '~/pages/validator/ValidatorDataTypes';
+
+import { Card } from '~/ui/Card';
+import { Heading } from '~/ui/Heading';
+import { Stats } from '~/ui/Stats';
+import { getStakedPercent } from '~/utils/getStakedPercent';
+
+type StatsCardProps = {
+    validatorData: Validator;
+    totalValidatorStake: bigint;
+    epoch: number;
+};
+
+export function ValidatorStats({
+    validatorData,
+    epoch,
+    totalValidatorStake,
+}: StatsCardProps) {
+    // TODO: add missing fields
+    // const numberOfDelegators = 0;
+    //  const selfStake = 0;
+    //  const lastEpoch = 0;
+    //  const totalRewards =  0;
+    //  const networkStakingParticipation = 0;
+    //  const votedLastRound =  0;
+    //  const tallyingScore =  0;
+    //  const lastNarwhalRound = 0;
+
+    const validator = useMemo(() => {
```

Comment:
As mentioned before, this is effectively a view model, and I'd avoid this and just do field computation as-needed. e.g....
```typescript
const APY = useMemo(() => /* */);
const totalState = validatorData.fields.stakeAmount;
const delegatedStakePercentage = getStakedPercent();
```

Reply:
Done